### ISSUE TITLE
Question on object's name

### ISSUE THREAD

**Issue by WayneCV** [czq142857/IM-NET#25]
Greets, I have another question: In your provide dataset (13 categories on ShapeNet), the training data are put together in one file `all_vox256_img_train.hdf5`. And I see the object's name in another file `all_vox256_img_train.txt`. Whether the object's name (in `all_vox256_img_train.txt`) corresponds to the data in  `all_vox256_img_train.hdf5` in order? For example, the data from the first line in `all_vox256_img_train.txt` corresponds to the first object from `all_vox256_img_train.hdf5`. Thanks in advance

**Comment by czq142857** [comment]
> Whether the object's name (in all_vox256_img_train.txt) corresponds to the data in all_vox256_img_train.hdf5 in order?

Yes. They are corresponded. That's the purpose of *all_vox256_img_train.txt*.

**Comment by WayneCV** [comment]
> > Whether the object's name (in all_vox256_img_train.txt) corresponds to the data in all_vox256_img_train.hdf5 in order?
> 
> Yes. They are corresponded. That's the purpose of _all_vox256_img_train.txt_.

Thanks!
